components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -448,74 +448,71 @@ union SwapTransactionParamsUnion {
 
 union SwapQuoteUnion {
   JupiterQuote jupiter_quote;
-  ZeroExQuote zero_ex_quote;
+  ZeroExQuoteInfo zero_ex_quote;
```

Comment:
should we change this to `zero_ex_quote_info` to distinguish from `ZeroExQuote`? it seems `ZeroExQuoteInfo` now contains an optional of `ZeroExQuote`

Reply:
The refactoring [here](https://github.com/brave/brave-core/pull/26231/commits/1bbee57c1c8414b6fb3fe409da02134df3741f56) should fix this.